clang/lib/CIR/CodeGen/CIRGenStmt.cpp

```diff
@@ -81,6 +83,16 @@ mlir::LogicalResult CIRGenFunction::emitSimpleStmt(const Stmt *s,
   return mlir::success();
 }
 
+mlir::LogicalResult CIRGenFunction::emitDeclStmt(const DeclStmt &s) {
+  assert(builder.getInsertionBlock() && "expected valid insertion point");
+
+  for (const auto *I : s.decls()) {
```

Comment:
single-line curleys, also, 
```suggestion
  for (const Decl *I : s.decls()) {
```